WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -236,14 +236,6 @@ character_event = {
 	
 	trigger = {
 		NOT = { year = 800 } # The migration period ended around 800
-		OR = { # Possible target religion groups
-			religion_group = christian
-			religion = finnish_pagan
-			religion = germanic_pagan
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```

Comment:
The easy answer would be: because we decided by consensus. 

The longer answer is that it causes AIs that had settled to migrate, adds randomness (no tribe migrated out of the blue), balkanises the map and goes against historical reasons and conditions behind migration. 

I disagree on Lunatics.